lib/TH/generic/THTensorMath.c

```diff
@@ -709,6 +709,46 @@ void THTensor_(baddmm)(THTensor *result, real beta, THTensor *t, real alpha, THT
   THTensor_(free)(matrix2);
 }
 
+void THTensor_(baddbmm)(THTensor *result, real beta, THTensor *t, real alpha, THTensor *batch1, THTensor *batch2)
```

Comment:
We could implement torch.bmm in terms of baddbmm, couldn't we? We just need to make sure that torch.bmm resizes the result tensor before passing it to badbmm.